cni/network/network_test.go

```diff
@@ -1084,9 +1084,21 @@ func TestGetNetworkName(t *testing.T) {
 	}
 }
 
-func TestGetOverlayNatInfo(t *testing.T) {
+func TestGetV4OverlayNatInfo(t *testing.T) {
 	nwCfg := &cni.NetworkConfig{ExecutionMode: string(util.V4Swift), IPAM: cni.IPAM{Mode: string(util.V4Overlay)}}
 	natInfo := getNATInfo(nwCfg, nil, false)
+	require.Empty(t, natInfo, "v4overlay natInfo should be empty")
+}
+
+func TestGetDualStackOverlayNatInfo(t *testing.T) {
```

Comment:
I believe we don't call these NAT info funcs for dualstackoverlay so no need to add these. I would suggest keeping the original one